src/Stripe.net/Infrastructure/Public/StripeConfiguration.cs

```diff
@@ -176,6 +180,30 @@ public static JsonSerializerSettings DefaultSerializerSettings()
             };
         }
 
+        /// <summary>
+        /// Sets information about your application. The information is passed along to Stripe.
```

Comment:
So this is really about plugins/third-party libs here. But the wording of the description makes it look like any app should set this. Should we tweak it?